### ISSUE TITLE
"wal: error: No input specified."

### ISSUE THREAD

**Issue by Ducky6944** [Kharacternyk/fzwal#1]
When I try and run this I get a FZF preview window however it shows no options, although it says there are 220 and when I move up or down a line the preview window says:

```shell
wal: error: No input specified.
--backend, --theme, -i or -R are required.
```
<img width="1680" alt="Screen Shot 2020-07-13 at 10 20 10 PM" src="https://user-images.githubusercontent.com/12100960/87373096-067e7980-c557-11ea-868c-c01059bea722.png">

Edit:
`wal --theme | sed '1,/Light Themes/d;/Extra/,$d'` outputs as expected but `wal --theme | sed '1,/Light Themes/d;/Extra/,$d' | sed -E -e '/^\S/d' -e 's/^ - (\S*) ?(.*)$/\1/'` Just results in blank lines for me.

Edit:
Appears that it does not work with the prepackaged macOS version of sed. installing gnu-sed from brew and editing the script to use gsed resolves the issue. 

Edit:
Apparently gnu's version of sed differs from the BSD version of sed, which is what comes with macOS. I'm not exactly sure what breaks, but it is something in `'s/^ - (\S*) ?(.*)$/\1/'` this pattern. I am terrible with sed and regex in general, but here's a workaround that seems to work on both gnu and bsd versions of sed just in case any macOS users are looking. 

```shell
if [ -n "$1" ]; then
    IS_LIGHT=TRUE
    THEME=$(wal --theme |
            sed '1,/Light Themes/d;/Extra/,$d' |
            sed -e 's/ - //' -e '/:$/d' -e 's/(.*)//' |
            fzf --preview='wal -qetl --theme {} && wal --preview')
else
    THEME=$(wal --theme | 
      sed '/Light Themes/,$d' | 
      sed -e 's/ - //' -e '/:$/d' -e 's/(.*)//' |
      fzf --preview='wal -qet --theme {} && wal --preview')
fi
```

**Comment by Kharacternyk** [comment]
I indeed used a GNU-specific feature of `sed`. It wasn't even needed, it had been just left from the early versions of the script where it was useful. I've fixed it in 3208568, now the regex looks like this: `sed -e '/^\S/d' -e 's/ - //'`. It works on GNU/Linux and I believe that it does on MacOS, too, but it would be nice if you tested it, just in case.

BTW, how do you recolor GitHub? Is it a Firefox-specific extension?

**Comment by Ducky6944** [comment]
<del>Awesome, this works just as well.</del>
![3208568](https://github.com/Kharacternyk/fzwal/commit/32085684f23f77218d2cc88b41243a5c17a9a43e) leaves me with the actual Listing Name `Dark Themes:`, but works for `Light Themes:`. Other than this it works as expected. 

I am using ![stylus](https://github.com/openstyles/stylus/) with `Dracula Github` and `Material Dark Github` themes both turned on to theme github. 

Thanks for your work on this too. Very cool.
